change/@ni-nimble-blazor-74953c7b-7a79-4534-8482-c6e922fb406e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I don't see any source changes in nimble-blazor so this could be none (not a big deal since the components change will cause a release anyway, but at least signals no change in CHANGELOG and change only due to components diff)